quarkus/server/distribution/LICENSE

```diff
@@ -2050,12 +2050,6 @@ License (from POM): Apache License 2.0 - https://www.apache.org/licenses/LICENSE
 
 --------------------------------------------------------------------------------
 
-Group: org.bouncycastle Name: bcprov-jdk18on Version: 1.80
```

Comment:
bouncycastle/bc-prov is still a dependency of polaris-quarkus-server (see dependencies of the `runtimeClasspath` Gradle configuration), so the removals in LICENSE/NOTICE are wrong.